r/vignettes/install.Rmd

```diff
@@ -174,16 +174,19 @@ The intent is that `install.packages("arrow")` will just work and handle all C++
 dependencies, but depending on your system, you may have better results if you
 tune one of several parameters. Here are some known complications and ways to address them.
 
-## Package installed without C++ dependencies
+## Package failed to build C++ dependencies
 
-If you get an error like
+If you see a message like
 
 ```
-Cannot call io___MemoryMappedFile__Open(). See https://arrow.apache.org/docs/r/articles/install.html for help installing Arrow C++ libraries.
+------------------------- NOTE ---------------------------
+See https://arrow.apache.org/docs/r/articles/install.html
+for help installing Arrow C++ libraries
+---------------------------------------------------------
 ```
 
-for every `arrow` function you call,
-that means that installing the package failed to retrieve or build C++ libraries
+in the output when the package fails to install,
+that means that installation failed to retrieve or build C++ libraries
 compatible with the current version of the R package.
```

Comment:
A few lines below (though I can't seem to commend on them in the GH UI), we should probably suggest _either_ `arrow::install_arrow(verbose = TRUE)` or `Sys.setenv(ARROW_R_DEV=TRUE); install.packages("arrow")` or however the person was installing it, now that they won't have the shell packages to call `arrow::install_arrow()` from.